tests/Integration/Behaviour/Features/Context/SqlManagerFeatureContext.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Tests\Integration\Behaviour\Features\Context;
+
+use PrestaShop\PrestaShop\Core\Domain\SqlManagement\Query\GetDatabaseTableFieldsList;
+use PrestaShop\PrestaShop\Core\Domain\SqlManagement\DatabaseTableFields;
+use PrestaShop\PrestaShop\Core\Domain\SqlManagement\ValueObject\DatabaseTableField;
+use PrestaShop\PrestaShop\Core\Domain\SqlManagement\Command\AddSqlRequestCommand;
+use Behat\Gherkin\Node\TableNode;
+
+/**
+ * SqlManagerFeatureContext provides behat steps to perform actions related to prestashop SQL management
+ * and validate returned outputs
+ */
+class SqlManagerFeatureContext extends AbstractPrestaShopFeatureContext
+{
+    /**
+     * @When I request the database fields from table :tableName
+     */
+    public function getDatabaseTableFieldsList($tableName)
```

Comment:
I'm not sure the function is specific to SQL Manager: shouldn't be located in the Abstract class?